app/gzac/build.gradle

```diff
@@ -76,6 +76,7 @@ dependencies {
 
     // Database
     implementation group: 'org.postgresql', name: 'postgresql', version: postgresqlDriverVersion
+    implementation "mysql:mysql-connector-java"
```

Comment:
Can we do this, license-wise?

Reply:
what do you mean?

Reply:
It has been added in the past, then removed because of license issues, and now it's added again. I'm lost.

Reply:
I've removed it for now